apps/studio/.env.example

```
@@ -1,9 +1,9 @@
 # Created by Vercel CLI
-SANITY_API_DATASET="production"
+SANITY_API_DATASET="development"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)